scripts/after_platform_add.js

```diff
@@ -1,8 +1,13 @@
 var async = require('async');
 var spawn = require('./spawn');
 
+var cordova = 'cordova';
+if (process.platform === 'win32') {
```

Comment:
Should we move this into `scripts/spawn.js` so that any future hook that uses spawn will be windows compatible?